.gitignore

```diff
@@ -23,3 +23,5 @@ output/*
 htmlcov
 .coverage
 pipeline_vars.yml
+
+build/concourse/pipeline_var.yml
```

Comment:
Should this file end with a blank new line?

Reply:
I can add one in

Reply:
Aye. All files, by definition, should end with a newline, as unix format dictates/